### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by kornelski**
#163

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 authors = ["Michael Gattozzi <mgattozzi@gmail.com>"]
 description = "Pure Rust bindings to the Github API"
 name = "github-rs"
-version = "0.6.1"
+version = "0.7.0"
 license = "MIT"
 repository = "https://github.com/mgattozzi/github-rs.git"
```
Comment (brndnmtthws) [inline Cargo.toml]: Update this while you're in here.

File: Cargo.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 homepage = "https://github.com/mgattozzi/github-rs"
```
Comment (brndnmtthws) [inline Cargo.toml]: Ditto.

**Comment by Dylan-DPC** [comment]
@brndnmtthws that is being done in #168

**Comment by brndnmtthws** [comment]
@Dylan-DPC ah, didn't see that. Thanks.

**Pull request merged**